netclient/ncutils/netclientutils.go

```diff
@@ -532,3 +540,131 @@ func CheckWG() {
 		log.Println("running userspace WireGuard with " + uspace)
 	}
 }
+
```

Comment:
Is this the right package for these functions.  Given they are called in both netmaker and netclient, would it make more sense to put them in logic or functions?

Reply:
can't be avoided in current code base